packages/fetchai/connections/oef/connection.py

```diff
@@ -139,6 +139,8 @@ def __init__(
 
         self._threaded_pool = ThreadPoolExecutor(self.THREAD_POOL_SIZE)
 
+        self.aea_logger = logger
```

Comment:
can we use `logger` or does this clash with oef?

Reply:
it does clash